pkgs/by-name/ka/kakasi/package.nix

```diff
@@ -28,10 +31,18 @@ stdenv.mkDerivation rec {
     sha256 = "1qry3xqb83pjgxp3my8b1sy77z4f0893h73ldrvdaky70cdppr9f";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "avoid-an-implicit-declaration-of-exit";
```

Comment:
nit: We’d generally want patch filenames to end with `.patch` or `.diff`, though there’s no functional change here. (And I think in this case it would have been fine to omit it, as the original URL has an explanatory filename anyway.)

(Non‐blocking, will try to remember to merge even if this is not addressed.)